assets/interfaces/common.ts

```diff
@@ -86,4 +86,5 @@ export interface IDateFilter {
     default?: boolean,
     filter?: string;
     query?: string;
+    find?: any;
```

Comment:
is this used somewhere?

Reply:
yep here :- https://github.com/superdesk/newsroom-core/pull/1021/files#diff-48af75964c59c6cf135686fe342f79b0e2743556429529a579f8e2c42af4faf0R326

Reply:
that looks like a wrong type then, there it should be an array of IDateFilter which already has find method